speech-assets/built/SampleUtterances.txt

```diff
@@ -770,3 +786,11 @@ AggregationIntent {Aggregate} rows in {TableName} do we have where {ComparisonCo
 AggregationIntent {Aggregate} rows in {TableName} do we have where {ComparisonColumnOne} {ComparatorOne} {ColumnNumberOne} {BinaryLogicOpOne} {ComparisonColumnTwo} {ComparatorTwo} {ColumnValueTwo} {BinaryLogicOpTwo} {ComparisonColumnThree} {ComparatorThree} {ColumnNumberThree} {GroupBy} {GroupByColumn}
 AggregationIntent {Aggregate} rows in {TableName} do we have where {ComparisonColumnOne} {ComparatorOne} {ColumnNumberOne} {BinaryLogicOpOne} {ComparisonColumnTwo} {ComparatorTwo} {ColumnNumberTwo} {BinaryLogicOpTwo} {ComparisonColumnThree} {ComparatorThree} {ColumnValueThree} {GroupBy} {GroupByColumn}
 AggregationIntent {Aggregate} rows in {TableName} do we have where {ComparisonColumnOne} {ComparatorOne} {ColumnNumberOne} {BinaryLogicOpOne} {ComparisonColumnTwo} {ComparatorTwo} {ColumnNumberTwo} {BinaryLogicOpTwo} {ComparisonColumnThree} {ComparatorThree} {ColumnNumberThree} {GroupBy} {GroupByColumn}
+ClarifyAmbiguousTableIntent {TableName}
+ClarifyAmbiguousTableIntent I meant {TableName}
+ClarifyAmbiguousTableIntent I'm referring to {TableName}
+ClarifyAmbiguousTableIntent How about {TableName}
+ClarifyAmbiguousTableIntent {TableAndColumnName}
+ClarifyAmbiguousTableIntent I meant {TableAndColumnName}
+ClarifyAmbiguousTableIntent I'm referring to {TableAndColumnName}
+ClarifyAmbiguousTableIntent How about {TableAndColumnName}
```

Comment:
yeah i guess we could but that would be more parsing to save a few lines of utterances - both of these slot types already have corresponding custom slots. maybe in a future PR because combining table and column for what's in the where clauses we'll have to do more error handling